docs/source/models.rst

```diff
@@ -829,22 +822,23 @@ accepts the following data formats as input:
                   headers={"Content-type": "application/json"})
     response_json = json.loads(response.text)
     print(response_json)
-
+    
```

Comment:
Looks like we still have unnecessary whitespace. More than happy to push a commit to remove this - let me know!

Reply:
thanks for pushing the commit! I appreciate it, however there's still a test failing, let me know what I can do. I don't fully understand the error

Reply:
@shivp950 It looks like the r test is a bit flaky. I'll rerun the CI checks.

Reply:
@harupy thank you!

Reply:
@dbczumar and @harupy if this pr looks good to you, can you please approve it when you get the chance?